### DISCUSSION TITLE
GetData function does not work

### DISCUSSION THREAD

**Discussion by Rrosso95** [migueesc123/PowerBIRESTAPI#163] [General]
Please, I need help, this image appears when I want to import the GetData() function.
<img width="646" alt="Captura GetData" src="https://user-images.githubusercontent.com/77074885/235184077-a7efbe6d-ce4a-4593-81f4-876fbee44538.PNG">

**Comment by migueesc123** [comment]
Hey!
This is by design. You haven't invoked any functions.

Moving this as a discussion as this is not a bug.
